modules/elasticsearch-impl/src/main/java/org/opencastproject/elasticsearch/impl/AbstractElasticsearchIndex.java

```diff
@@ -284,6 +286,57 @@ protected IndexResponse update(int maxRetryAttempts, int retryWaitingPeriod, Ela
     return indexResponse;
   }
 
+  /**
+   * Posts the input documents to the search index.
+   *
+   * @param maxRetryAttempts
+   *          How often to retry update in case of ElasticsearchStatusException
+   * @param retryWaitingPeriod
+   *          How long to wait (in ms) between retries
+   * @param documents
+   *          The Elasticsearch documents
+   * @return the query response
+   *
+   * @throws IOException
+   *         If updating the index fails
+   * @throws InterruptedException
+   *         If waiting during retry is interrupted
+   */
+  protected BulkResponse bundleUpdate(int maxRetryAttempts, int retryWaitingPeriod,
+      List<ElasticsearchDocument> documents)
+          throws IOException, InterruptedException {
+    BulkRequest bulkRequest = new BulkRequest().setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE);
```

Comment:
It seems that opencast doesn't display the actual version of a new created event in the adminUI if we don't use NONE instead of IMMEDIATE as refresh policy. The meta data fields remain empty in this case even if the ES doc has been submitted. Than an complete index rebuild would be necessary.